Send all logs to concord_py.log

RBlafford opened the pull request:
- Also prevent all logs from going to stderr
- Display datetime in log formatter
- Increment version to 0.2.6

Commit:
Send all logs to concord_py.log

- Also prevent all logs from going to stderr
- Display datetime in log formatter
- Increment version to 0.2.6

ghost commented:
@senior7515 @bigs  not sure how I would expose this to the client. Like to hear everyones feedback.

Commit:
Changing logging formatter and user interface

- New formatter format: level, time, filename, lineno, log
- Removing unnecessary global client_logger
- Method concord_logger returns global logger to client
- Adding ivar to handler in 'serve_computation' so handler can
  use same logger (i.e. self.concord_logger)

Commit:
Removing duplicate code

Commit:
Renaming logger to ccord_logger

Commit:
Fixing setup.py for pypi distribution

- Adding requirements.txt
- Removing hardcoded requirements in setup.py
- Removing activate script

RBlafford commented on concord/computation.py:
```diff
@@ -239,13 +243,19 @@ def set_proxy_address(self, host, port):
     """
-    concord_logger.info("About to serve computation and service")
+    ccord_logger.info("About to serve computation and service")
+    if not 'logger' in dir(handler):
```
should be concord_logger

Commit:
Fixing bug where check for mvar will fail
